cicecore/cicedyn/infrastructure/ice_grid.F90

```diff
@@ -224,7 +224,7 @@ subroutine alloc_grid
          ocn_gridcell_frac(nx_block,ny_block,max_blocks),& ! only relevant for lat-lon grids
          hm       (nx_block,ny_block,max_blocks), & ! land/boundary mask, thickness (T-cell)
          bm       (nx_block,ny_block,max_blocks), & ! task/block id
-         uvm      (nx_block,ny_block,max_blocks), & ! land/boundary mask, velocity (U-cell) - water in case of all water point
+         uvm      (nx_block,ny_block,max_blocks), & ! land/boundary mask, velocity (U-cell) - all water point
```

Comment:
Sorry, but I don't understand this comment. Are we trying to say that uvm is only ice-covered points with aice > puny?

Reply:
I don't understand it either and didn't try to, just wanted to shorten the line.  I think it's a u mask where all u points are surrounded by non-land t points.  If someone wants to suggest an updated comment, I'm happy to change it.

Reply:
That comment came in with some of the C-grid stuff.  Please remove the 'water' part.  I think what it means is that uvm = 1 only when all of the surrounding cells are not land (which is the definition of uvm all the time).